docs/Options.md

```diff
@@ -180,86 +239,69 @@ Values: `true` or `false` | Default =  `false`
 
 Set uWSGI [log-x-forwarded-for](https://uwsgi-docs.readthedocs.io/en/latest/Options.html?#log-x-forwarded-for) parameter allowing to keep the real IP of clients in logs when using a reverse proxy.
 
-### `LD_DB_ENGINE`
-
-Values: `postgres` or `sqlite` | Default = `sqlite`
-
-Database engine used by linkding to store data.
-Currently, linkding supports SQLite and PostgreSQL.
-By default, linkding uses SQLite, for which you don't need to configure anything.
-All the other database variables below are only required for configured PostgresSQL.
-
-### `LD_DB_DATABASE`
 
-Values: `String` | Default =  `linkding`
+## Single File Options
 
-The name of the database. 
 
-### `LD_DB_USER`
+### `LD_ENABLE_SNAPSHOTS`
 
-Values: `String` | Default =  `linkding`
+Values: `Bool` | Default = True
 
-The name of the user to connect to the database server.
+This enables the single-file snapshot functionality. When using the docker images, this value is set in the dockerfile build script.
 
-### `LD_DB_PASSWORD`
 
-Values: `String` | Default =  None
+### `LD_SINGLEFILE_OPTIONS`
 
-The password of the user to connect to the database server.
-The password must be configured when using a database other than SQLite, there is no default value.
+Values: `String` | Default = None
 
-### `LD_DB_HOST`
+When creating HTML archive snapshots, pass additional options to the `single-file` application that is used to create snapshots.
+See `single-file --help` for complete list of arguments, or browse source: https://github.com/gildas-lormeau/single-file-cli/blob/master/options.js
 
-Values: `String` | Default =  `localhost`
+Example: `LD_SINGLEFILE_OPTIONS=--user-agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0"`
 
-The hostname or IP of the database server.
 
-### `LD_DB_PORT`
+### `LD_SINGLEFILE_PATH`
```

Comment:
This probably shouldn't be changed either. If there's no use case for changing this it seems better to remove it to reduce confusion for readers what they should do with it.